### PR TITLE
Add missed $searchCriteriaSearch property

### PR DESCRIPTION
**Pull request by thicolares**
### Description (*)

- Add missed $searchCriteriaSearch property
- Analyse against phpstan returned ""[OK] No errors"

### Fixed Issues (if relevant)

1. magento-engcom/import-export-improvements#127: Clean up Magento/CatalogImportExport/Test/Unit/Model/Import/Product/Validator/TierPriceTest

### Manual testing scenarios (*)

1. Run ./vendor/bin/phpstan analyse -l 0 app/code/Magento/CatalogImportExport/Test/Unit/Model/Import/Product/Validator/TierPriceTest.php

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [X] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by dmanners** [comment]
Hello, thank you for your pull request. I will start to process this PR and get back to you if I need any more information.

**Comment by thicolares** [comment]
Ok! I am glad to help

File: app/code/Magento/CatalogImportExport/Test/Unit/Model/Import/Product/Validator/TierPriceTest.php [hunk 1]
```diff
@@ -19,6 +19,9 @@ class TierPriceTest extends \PHPUnit\Framework\TestCase
     /** @var \Magento\Customer\Api\GroupRepositoryInterface|\PHPUnit_Framework_MockObject_MockObject */
     protected $groupRepositoryInterface;
 
+    /** @var \Magento\Framework\Api\SearchCriteria|\PHPUnit_Framework_MockObject_MockObject */
+    protected $searchCriteriaSearch;
```
Comment (dmanners) [inline app/code/Magento/CatalogImportExport/Test/Unit/Model/Import/Product/Validator/TierPriceTest.php:23]: I think looking at it's usage we do not need this as a call variable but could make it local to the method only. Would you be up for making this change and then validating the phpstan and phpunit test again?

Comment (thicolares) [reply app/code/Magento/CatalogImportExport/Test/Unit/Model/Import/Product/Validator/TierPriceTest.php:23]: Sure! Besides `$searchCriteriaSearch`, I believe I should do the same for `$objectManagerHelper` and `$searchCriteriaBuilder` -- both are also only local and not instantiated via dependency injection.

**Comment by magento-engcom-team** [comment]
@thicolares thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by dmanners** [comment]
Thank you for your effort here. As we are focusing import work on the async-import project we will be closing this project for now.

**Pull request closed without merging**